src/_locales/ru/messages.json

```
@@ -60,76 +60,76 @@
-    "html_options_other_enable_sync_tooltip_line1b": { "message": "с тем же профилем." },
-    "html_options_other_enable_sync_tooltip_line2": { "message": "Это также синхронизирует настройки в режиме инкогнито." },
-    "html_options_other_enable_sync_warning": { "message": "Включение этого будет перезаписывать настройки во всех других браузерах, зарегистрированных в том же профиле Chrome" },
-    "html_options_suspend_on_low_memory": { "message": "Мгновенно приостанавливается при нехватке системной памяти" },
-    "html_options_suspend_on_low_memory_tooltip_line1": {"message": "Этот параметр будет отменять автоматическую отмену функциональных возможностей браузера Chrome." },
-    "html_options_suspend_on_low_memory_tooltip_line2": {"message": "Если включено, это заставит вкладку приостанавливаться, а не отбрасываться Chrome." },
-    "html_options_suspend_on_low_memory_tooltip_line3": {"message": "Как правило, Chrome будет только отбрасывать вкладки, когда системная память становится очень низкой." },
-    "html_options_suspend_on_low_memory_tooltip_line4": {"message": "Обратите внимание: это может привести к приостановке вкладок раньше, чем у вас есть"},
-    "html_options_suspend_on_low_memory_tooltip_line5": {"message": "указано в параметрах автоматической приостановки табуляции." },
-    "html_options_suspend_on_low_memory_tooltip_line6": {"message": "Для получения дополнительной информации об отбрасывании вкладок Chrome см.:"},
+    "html_options_other_enable_context_menu_tooltip_line2": { "message": "Это даст Вам доступ к дополнительным действиям с вкладками, которые" },
+    "html_options_other_enable_context_menu_tooltip_line3": { "message": "недоступны в другом месте расширения." },
+    "html_options_other_enable_sync": { "message": "Синхронизировать настройки с Вашим профилем Chrome" },
+    "html_options_other_enable_sync_tooltip_line1a": { "message": "Настройки расширения будут синхронизированы с Вашим профилем между" },
+    "html_options_other_enable_sync_tooltip_line1b": { "message": "всеми браузерами Chrome." },
+    "html_options_other_enable_sync_tooltip_line2": { "message": "Также синхронизирует настройки в режиме инкогнито." },
+    "html_options_other_enable_sync_warning": { "message": "Включение этой опции приведет к перезаписыванию настроек расширения для Вашего профиля между всеми браузерами Chrome" },
+    "html_options_suspend_on_low_memory": { "message": "Приостанавливать вкладки при нехватке системной памяти" },
+    "html_options_suspend_on_low_memory_tooltip_line1": {"message": "Этот параметр заменяет встроенный в Chrome механизм выгрузки вкладок." },
+    "html_options_suspend_on_low_memory_tooltip_line2": {"message": "Если включено, вкладка будет принудительно приостанавливаться, вместо использования механизма выгрузки Chrome." },
+    "html_options_suspend_on_low_memory_tooltip_line3": {"message": "Как правило, Chrome выгружает вкладки, только когда системной памяти становится очень мало." },
```

"когда доступной оперативной памяти остаётся очень мало" is better, I think.

---

Yep, I support this one, sounds really better

---

i use "системной", as here https://developers.google.com/web/updates/2015/09/tab-discarding
in English "system" == "ram"? in Russian it can mean any type of memory (hdd, ram, cache etc )

---

> i use "системной", as here https://developers.google.com/web/updates/2015/09/tab-discarding

Thank you. But there is `system` only in 2 of 13 cases, in others there is just "memory", without any adjective.

But OK, "system" is acceptable, and it's in the English locale.

So, with other changes here will be:

```suggestion
    "html_options_suspend_on_low_memory_tooltip_line3": {"message": "Как правило, Chrome выгружает вкладки, только когда системной памяти остаётся очень мало." },
```

But I think, you can understand my confusion: without clarification there can be any system memory: used, available, free, general, etc. But it's also a problem of English version.

/cc @deanoemcke

---

I think that after all there is an option only for "RAM", so I replaced the word "system".
fixed

---

And what bout the word "доступной" then, do we need it, @AlexWayfer?
My opinion, it's ok now with the latest @MonsterNya's change, without it.

---

> I think that after all there is an option only for "RAM", so I replaced the word "system".

No, there is no "RAM" in the article by Google. What about swap? DRAM?

> And what bout the word "доступной" then, do we need it, @AlexWayfer?

Only when the English version will contain it.

---

> > I think that after all there is an option only for "RAM", so I replaced the word "system".
> 
> No, there is no "RAM" in the article by Google. What about swap? DRAM?
> 
So instead of "системной" (system) which word should be used in russian translation then, if not "оперативной" (ram)? What's ur point here? And I think most of users in general will barely even understand what "оперативной" (ram) is, not speaking of swap, DRAM, etc.. Soo imo word "оперативной" (ram) is better than "системной" (system) and already enough for who knows the basics, no need to complicate even more.

> > And what bout the word "доступной" then, do we need it, @AlexWayfer?
> 
> Only when the English version will contain it.
> 
Alright then, at least here no changes needed for now.

---

> So instead of "системной" (system) which word should be used in russian translation then, if not "оперативной" (ram)?

I think "системной" is correct translation for "system" from English locale.

> What's ur point here?

Translate correctly.

> And I think most of users in general will barely even understand what "оперативной" (ram) is, not speaking of swap, DRAM, etc..
> Soo imo word "оперативной" (ram) is better than "системной" (system) and already enough for who knows the basics, no need to complicate even more.

But neither Google Blog nor English locale of this extension used "RAM".

> Alright then, at least here no changes needed for now.

Changes are needed. Here is "оперативная", which is wrong translation for "system".

---

By changes I meant for word "доступной", to add it, but we decided not to, so

> at least here no changes needed for now

sorry for my English, I guess 🙄

And imo we should use "оперативной" (ram) as it is now, I think we may even ask @deanoemcke  what he meant by "system memory" as we should not translate literally, was it ram? or anything else? coz if ram we should go with "оперативной" and that's it.

---

Oh. thinking about this again, yet more appropriate "system". system = ram + swop (+ hdd) ...
although in both cases it is not an accurate list.